src/main/kotlin/no/nav/familie/ef/sak/vedtak/uttrekk/OpprettUttrekkArbeidssøkerTask.kt

```diff
@@ -15,11 +19,36 @@ import java.time.YearMonth
 )
 class OpprettUttrekkArbeidssøkerTask(
         private val uttrekkArbeidssøkerService: UttrekkArbeidssøkerService,
+        private val fagsakService: FagsakService,
         private val taskRepository: TaskRepository
 ) : AsyncTaskStep {
 
+    private val logger = LoggerFactory.getLogger(javaClass)
+
     override fun doTask(task: Task) {
-        uttrekkArbeidssøkerService.opprettUttrekkArbeidssøkere()
+        val årMåned = objectMapper.readValue<YearMonth>(task.payload)
+        val uttrekk = uttrekkArbeidssøkerService.hentArbeidssøkereForUttrekk(årMåned)
+        val aktiveIdenter = fagsakService.hentAktiveIdenter(uttrekk.map { it.fagsakId }.toSet())
+
+        var feilede = 0
+        uttrekk.forEach {
+            if (uttrekkArbeidssøkerService.uttrekkFinnes(årMåned, it.fagsakId)) {
+                return@forEach
+            }
+            try {
+                uttrekkArbeidssøkerService.opprettUttrekkArbeidssøkere(årMåned = årMåned,
+                                                                       fagsakId = it.fagsakId,
+                                                                       behandlingIdForVedtak = it.behandlingIdForVedtak,
+                                                                       personIdent = aktiveIdenter[it.fagsakId]
+                                                                                     ?: error("Kunne ikke finne fagsakID"))
+            } catch (ex: Exception) {
+                logger.error(ex.message, ex)
```

Comment:
Kan vi kanskje gjøre noe slik for å unngå å logge fnr i loggen? eks
Mangler en del context her også tenker jeg
```suggestion
                val errorMelding = "Sjekk av utrekkArbeidssøker feiler fagsak=$it.fagsakId behandling=$it.behandlingId"
                logger.error(errorMelding)
                secureLogger.error("$errorMelding - ${ex.message}", ex)
```